scripts/inpage-bridge/content-script/index.js

```diff
@@ -109,27 +109,31 @@ function blockedDomainCheck() {
     'uscourts.gov',
     'dropbox.com',
     'webbyawards.com',
-    'cdn.shopify.com/s/javascripts/tricorder/xtld-read-only-frame.html',
     'adyen.com',
     'gravityforms.com',
     'harbourair.com',
     'ani.gamer.com.tw',
     'blueskybooking.com',
     'sharefile.com',
-  ];
-  const currentUrl = window.location.href;
-  let currentRegex;
-  for (let i = 0; i < blockedDomains.length; i++) {
-    const blockedDomain = blockedDomains[i].replace('.', '\\.');
-    currentRegex = new RegExp(
-      `(?:https?:\\/\\/)(?:(?!${blockedDomain}).)*$`,
-      'u',
-    );
-    if (!currentRegex.test(currentUrl)) {
-      return true;
-    }
-  }
-  return false;
+  ]
+
+  const blockedHrefs = [
+    'cdn.shopify.com/s/javascripts/tricorder/xtld-read-only-frame.html',
+  ]
+
+  try {
+    const currentUrl = new URL(window.location.href);
+    const hostname = currentUrl.hostname;
+    const path = currentUrl.pathname;
+    
+    const blockedDomainFound = blockedDomains.some(blockedDomain => hostname.endsWith(blockedDomain));
```

Comment:
Nit: While this is an improvement over the previous version, this might block too many hostnames. For example if `bar.com` was on the blocklist, this would block `foobar.com` as well despite having no connection.

Maybe better to do something like this:

```suggestion
    const blockedDomainFound = blockedDomains.some(blockedDomain => blockedDomain === hostname || hostname.endsWith(`.${blockedDomain}`));
```